src/locale/messages.fr.xlf

```diff
@@ -500,15 +495,15 @@
         <note priority="1" from="description">faq answer</note>
         <target state="translated">Votre domaine n&apos;est pas encore délégué.</target>
       </trans-unit>
-      <trans-unit id="5bad6b87847399d805291b8a8d89a84ef6576c42" datatype="html">
-        <source>Lastly, this open source version of Zonemaster was built using modular code which basically means that you can integrate parts of it in your own systems, if you wish. For example, it is quite rare that you would want a complete program just to check for redelegations.</source>
+      <trans-unit id="865aac49d53d372537909b1df21a9d111d50c9f1" datatype="html">
+        <source>This open source version of Zonemaster was built using modular code which basically means that you can integrate parts of it in your own systems, if you wish. For example, it is quite rare that you would want a complete program just to check for redelegations.</source>
         <note priority="1" from="description">faq answer</note>
-        <target state="new">Enfin, cette version « open source » de Zonemaster a été écrite de manière modulaire, ce qui signifie en résumé qu&apos;il est possible d&apos;intégrer des parties de Zonemaster dans vos propres systèmes, si vous le souhaitez. Par exemple, on souhaite rarement utiliser un logiciel complet juste pour vérifier des redélégations.</target>
+        <target state="translated">Enfin, cette version « open source » de Zonemaster a été écrite de manière modulaire, ce qui signifie en résumé qu&apos;il est possible d&apos;intégrer des parties de Zonemaster dans vos propres systèmes, si vous le souhaitez. Par exemple, on souhaite rarement utiliser un logiciel complet juste pour vérifier des redélégations.</target>
```

Comment:
Because “lastly” was removed in the source, the same should be done in the target:
```suggestion
        <target state="translated">Cette version « open source » de Zonemaster a été écrite de manière modulaire, ce qui signifie en résumé qu&apos;il est possible d&apos;intégrer des parties de Zonemaster dans vos propres systèmes, si vous le souhaitez. Par exemple, on souhaite rarement utiliser un logiciel complet juste pour vérifier des redélégations.</target>
```

Reply:
Fixed.